todo_app/app.py

```diff
@@ -1,20 +1,23 @@
 from flask import Flask, render_template, request, redirect
-
 from todo_app.flask_config import Config
-
-from todo_app.data.session_items import get_items, add_item
+from todo_app.data.trello_items import get_done_items, get_to_do_items, add_item, complete_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    items = get_items()
-    return render_template('index.html', items=items)
+    to_do_items = get_to_do_items()
+    done_items = get_done_items()
+    return render_template('index.html', to_do_items=to_do_items, done_items=done_items)
 
 @app.route('/submit', methods=['POST'])
 def addItem():
     newItem = request.form.get('title')
     add_item(newItem)
+    return redirect('/')
+
+@app.route('/completeItem<id>')
+def completeItem(id):
```

Comment:
You should stick to the Python naming convention: snake_case (so `complete_item`) for variable / function names. If you were avoiding the name clash, you could call this `complete_item_route`

Also e.g. `new_item` above.